test/utilities_test.cpp

```diff
@@ -0,0 +1,64 @@
+//
+// test/utilities_test.cpp
+// PSim
+//
+// Contributors:
+//   Kyle Krol  kpk63@cornell.edu
+//
+// Pathfinder for Autonomous Navigation
+// Space Systems Design Studio
+// Cornell Univeristy
+//
+
+#include <lin.hpp>
+#include <gnc_utilities.hpp>
+
+#include <gtest/gtest.h>
+
+#define ASSERT_VEC_NEAR(a, b) ASSERT_NEAR(lin::fro(a - b), 0.0, 1e-4)
+
+using namespace gnc;  // Only b/c this repo pretty much defines all of ::gnc
+
+TEST(UtilitiesTest, QuaternionConjugateTest) {
+  lin::Vector4f q = {2.0f, 1.0f, -2.5f, 1.0f};
```

Comment:
Might want to also test near-zero values to make sure output quaternions don't go NaN? I don't know enough quaternion math yet to know if/how this might happen, but wanted to make the suggestion.

Reply:
I think we are pretty well guarded against that for the time being. Especially when it comes to the functions added in this PR - there isn't any scalar division stuff that could result in `NaN`.